src/main/java/org/hisp/dhis/rules/Option.java

```diff
@@ -0,0 +1,54 @@
+package org.hisp.dhis.rules;
+
+
+/**
+ * @author rajazubair
+ */
+
+public class Option
+{
+    private String name;
+
+    private String code;
+
+    public Option(String name, String code) {
+        this.name = name;
+        this.code = code;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getCode() {
+        return code;
+    }
+
+    public static Option.Builder builder()
```

Comment:
Do we really need this builder?
I prefer to use the constructor and to have name and code has final so we have an Immutable object